components/sup/src/manager/service/config.rs

```diff
@@ -478,93 +478,32 @@ pub struct Pkg {
     pub svc_files_path: String,
     pub svc_static_path: String,
     pub svc_var_path: String,
-    pub svc_user: Option<String>,
-    pub svc_group: Option<String>,
-    pub svc_user_default: String,
-    pub svc_group_default: String,
+    pub svc_user: String,
```

Comment:
Much nicer, only the *used* values and not optional--there is always a set value.